src/lib.rs

```diff
@@ -212,6 +212,18 @@ impl Builder {
         self
     }
 
+    /// Whether to use the clang-provided name mangling. This is true and
+    /// probably needed for C++ features.
+    ///
+    /// However, some old libclang versions seem to return incorrect results in
+    /// some cases for non-mangled functions, see [1], so we allow disabling it.
+    ///
+    /// [1]: https://github.com/servo/rust-bindgen/issues/528
```

Comment:
Nitpick: when rendered as markdown, I think this link is going to disappear, since the "see [1]" isn't a proper markdown link. Instead it should be something like

> ... [old libclang versions seem to return incorrect results][1] ...

above.